Number_types/include/CGAL/boost_mp.h

```diff
@@ -139,6 +141,258 @@ struct Algebraic_structure_traits<boost::multiprecision::detail::expression<T1,T
 
 // Real_embeddable_traits
 
+namespace Boost_MP_internal {
+
+  inline
+  Interval_nt<false> shift_positive_interval( const Interval_nt<false>& intv, const int e ) {
```

Comment:
Could you document the preconditions for this function? If we only assume that the interval is positive, the code seems wrong. With more assumptions maybe... :man_shrugging: